main.go

```diff
@@ -62,6 +64,19 @@ var podCommand = &cobra.Command{
 	},
 }
 
+var buildCommand = &cobra.Command{
+	Use: "build",
+	Run: func(cm *cobra.Command, args []string) {
+		fmt.Println("Building image from docker file at source")
+		err := cmd.Build(envVar, deployVar)
+		if err == nil {
+			fmt.Println("Build Over")
```

Comment:
This command doesn't trigger a build yet. Could you fix the message accordingly?

https://github.com/cli-playground/kodo/issues/23 tracks auto-triggering of the build.

Reply:
Shall make the change

Reply:
Message has been changed to `BuildConfig and ImageStrem Created Successfully, Build can now be Started`